### ISSUE TITLE
Stepper "summary" animation and overflow/height bugs 

### ISSUE THREAD

**Issue by smurfy** [Teradata/covalent#1312]
#### Bug Report

There are two bugs with the Stepper related to summary. Possible related.
Both can be observed here (Completed State step):
https://teradata.github.io/covalent/#/components/steps

1. Animation bug

Clicking on "Completed State" causes "Basic usage" content to close which is not smooth and jumps/jitters.
Same behavior can be observed by clicking multiple times on  "Completed State" to toggle between summary and content.

2. Overflow/Height bug

Can be seen on  "Completed State"'s summary.  There is a scrollbar but no content if i scroll down.
My guess is that summary somehow inherits height of content, because in my own project the white space after the actual summary <span> is much higher 

#### What is the expected behavior?

No jumping and jitter, correct height without scrollbars of summary

#### Which version of Angular and Material, and which browser and OS does this issue affect?

Angular 7.1.1 and Covalent 2.0.0-beta.5

**Comment by felipeik** [comment]
Hi, I just investigated this issue and it seems that both bugs were fixed in Covalent v2.0.0-rc.1

The https://teradata.github.io/covalent/#/components/steps link is already fixed too.

I'm not sure, but I think it is listed in the CHANGELOG:

```
BUG FIXES
animations: changing to overflow hidden instead of visibility hidden
```

**Comment by smurfy** [comment]
@emoralesb05 2.0.0-rc.1 fixes the problem, thank you.
